test/test_lava_test_plans.py

```diff
@@ -11,7 +11,10 @@
 
 # all tests all devices
 devices = [os.path.basename(d) for d in glob.glob("lava_test_plans/devices/*")]
-testcases = [os.path.basename(d) for d in glob.glob("lava_test_plans/testcases/*.yaml")]
+testcases = [ os.path.basename(d) \
+                for d in glob.glob("lava_test_plans/testcases/*.yaml") \
+                    if not d.startswith("lava_test_plans/testcases/android-")
+            ]
```

Comment:
maybe like this

-testcases = [os.path.basename(d) for d in glob.glob("lava_test_plans/testcases/*.yaml")]
+testcases = [os.path.basename(d) for d in glob.glob("lava_test_plans/testcases/[!android-]*.yaml")]

Reply:
it works with "[!android-]*.yaml"", but I could not read it as [] are normally used for ranges:(